cfme/fixtures/automate.py

```diff
@@ -34,3 +34,32 @@ def klass(namespace):
     )
     yield klass
     klass.delete_if_exists()
+
+
+@pytest.fixture(scope="module")
+def request_cls(domain):
+    """custom request class"""
+    original_class = (
+        domain.parent.instantiate(name="ManageIQ")
+        .namespaces.instantiate(name="System")
+        .classes.instantiate(name="Request")
+    )
+    original_class.copy_to(domain=domain)
+    yield domain.namespaces.instantiate(name="System").classes.instantiate(name="Request")
```

Comment:
I don't like that I don't see where are the object that the fixtures created being deleted.

Reply:
@jaryn it just compy `Request` to newly created domain